Products/PluggableAuthService/plugins/tests/test_ZODBGroupManager.py

```diff
@@ -78,6 +91,20 @@ def test_addGroup(self):
         info = info_list[0]
         self.assertEqual(info['id'], 'group')
 
+    def test_addGroup_CreatedEvent(self):
+        provideHandler(groupCreatedHandler)
```

Comment:
This should be done in `setUp` and undone in `tearDown`.

Reply:
@icemac mimics test_UserFolder.UserEvents.testUserCreationEvent...

I adapted code, thank you for review.

Gauthier